give-paymill.php

```diff
@@ -20,7 +20,7 @@
 	define( 'GIVE_PAYMILL_VERSION', '1.1' );
 }
 if ( ! defined( 'GIVE_PAYMILL_MIN_GIVE_VERSION' ) ) {
-	define( 'GIVE_PAYMILL_MIN_GIVE_VERSION', '1.8.3' );
+	define( 'GIVE_PAYMILL_MIN_GIVE_VERSION', '2.0.0' );
```

Comment:
@raftaar1191 why we can not update this to `2.1.0`

Reply:
Yes, @ravinderk we can do that but if this add-on minimum requirement is Give Core 2.0.0 then why would we updated it to 2.1.0


Example:
There may be the user who does not want to update to Give Core 2.1 but if they are using this Add-on then they have to forcefully update there Give Core to 2.1. 

So indirectly this addon works with Give Core 2.0 as well as with Give Core 2.1, So I guess we do not need to update the the Give Minimum version requirement to 2.1